lib/h-include.js

```diff
@@ -107,17 +103,32 @@ window.HInclude.HIncludeElement = window.HIncludeElement = (function() {
     }
   };
 
-  var include = function(element, url, includeCallback) {
-    var alternativeUrl = element.getAttribute('when-false-src');
-    if(!element.conditionalInclusion.call(element)) {
-      // if alt is specified, set url to alt if the predicate fails
-      if(alternativeUrl) {
-        url = alternativeUrl;
-      } else {
+  var include = function(element, includeCallback) {
+    var url = element.getAttribute('src');
+    var whenFalseUrl = element.getAttribute('when-false-src');
+    var altUrl = element.getAttribute('alt');
+
+    // Skip conditionalInclusion if performing an altSrcInclude pass
```

Comment:
Could you extract a pure function here that takes strings and returns the url to use?

Reply:
Latest update now contains this function that we could put somewhere else and test, a separate file?

```javascript
var getConditionalUrl = function(element, whenCondition, mediaCondition, altSrcInclude) {
  var url = element.getAttribute('src');
  var whenFalseUrl = element.getAttribute('when-false-src');
  var altUrl = element.getAttribute('alt');

  if(altSrcInclude) {
    url = altUrl;
  }
  else {
    if(whenCondition) {
      url = whenFalseUrl;
    }
    if(mediaCondition) {
      url = null;
    }
  }
  return url;
};
```